.github/ISSUE_TEMPLATE/documentation.yaml

```
@@ -0,0 +1,40 @@
+name: Documentation
```

https://github.com/euphony-io/euphony/issues/new/choose
Bug report has a title with emoji. I think alignment of title looks good.

```suggestion
name: 📚 Documentation
```